src/Resource/Payment.php

```diff
@@ -373,4 +400,64 @@ public function setOrder(Orders $order)
 
         return $this;
     }
+
+    /**
+     * Turns on a delay on credit card payment capture (pre-authorization).
+     *
+     * @return void
+     */
+    public function delayCapture()
```

Comment:
Renomeie esse método para `setDelayCapture` e faça ele retornar `$this`
Isso fará com que, quem já utilize o método não tenha falhas em sua integração.